python_modules/automation/automation/docker/image_defs.py

```diff
@@ -197,7 +197,7 @@ def dagster_celery_k8s_editable_cm(cwd: str) -> Iterator[None]:
 
 # Some images have custom build context manager functions, listed here
 CUSTOM_BUILD_CONTEXTMANAGERS: Dict[str, Callable] = {
-    "k8s-dagit-example": k8s_webserver_example_cm,
```

Comment:
[1] is this safe to change? didn't we end up having to keep the `dagit` image builds?

Reply:
good point, let's leave the name here (I support removing this in a separate PR)